.circleci/config.yml

```
@@ -25,10 +25,12 @@ jobs:
       - run:
           name: install php extensions
           command: |
-            sudo pecl install mongodb redis
-            sudo apt-get install libpng-dev libjpeg-dev
+            sudo apt-get install libpng-dev libjpeg-dev libmagickwand-dev --no-install-recommends
+            (yes '' || true) | sudo pecl install mongodb
+            (yes '' || true) | sudo pecl install redis
+            (yes '' || true) | sudo pecl install imagick
```

The `yes ''` command repeatedly pipes a newline character, which auto-selects any prompts from `pecl install` (which is necessary there's no `--non-interactive` option to avoid prompts, and so those would otherwise just wait forever and time out).

The `yes` command just keeps printing the same "enter key" to the console indefinitely, so by default this fails with an error code 147 due to `pecl install` not consuming all of the provided input. We can use the `|| true` on `yes` to swallow up that error code from `yes` while still allowing errors from `pecl install` to fail the build.

---

I've also split out the `pecl install` steps into separate lines based on [the documentation for this Docker box](https://hub.docker.com/_/php). Otherwise, any errors installing any but the last extension are just ignored! 😯

---

(They also suggest pinning exact extension versions, which I don't think we want to do here since that means more overhead for us to make sure we update these over time. If we ever end up installing a version that isn't compatible with our PHP version, we can pin then!)

---

Whoa. But who doesn't appreciate an agreeable config ☺️